ipn/ipnlocal/peerapi.go

```diff
@@ -626,41 +631,92 @@ func (h *peerAPIHandler) peerHasCap(wantCap tailcfg.PeerCapability) bool {
 	return h.ps.b.PeerCaps(h.remoteAddr.Addr()).HasCapability(wantCap)
 }
 
-func (h *peerAPIHandler) handlePeerPut(w http.ResponseWriter, r *http.Request) {
-	if !h.canPutFile() {
-		http.Error(w, "Taildrop access denied", http.StatusForbidden)
-		return
+var errMisconfiguredInternals = errors.New("misconfigured internals")
```

Comment:
We don't reference this elsewhere other than L639, should we be?

Reply:
No that's the only place we use that specific error message, we could always use something more generic if desired or I could just inline it.